package.json

```diff
@@ -95,7 +95,5 @@
     "recursive-readdir": "^2.2.1",
     "unist-util-map": "^1.0.3"
   },
-  "resolutions": {
-    "**/sharp": "^0.30.3"
-  }
+  "packageManager": "yarn@3.2.1"
```

Comment:
This is creating a large diff in the lockfile. This project requires yarn v1. If you use that for install you should not have any changes to the lockfile.

Reply:
> This is creating a large diff in the lockfile. This project requires yarn v1. If you use that for install you should not have any changes to the lockfile.

Sorry, I think this error might happened while running `next:lint` :(